lib/chef/provider/package/chocolatey.rb

```diff
@@ -160,9 +160,9 @@ def choco_exe
 
         # lets us mock out an incorrect value for testing.
         def choco_install_path
-          @choco_install_path ||= powershell_out!(
+          @choco_install_path ||= powershell_exec!(
```

Comment:
Also it looks like there's no need to memoize this since it just goes into the `choco_exe` variable which is memoized